gauntlet/packages/gauntlet-solana-contracts/src/commands/contracts/ownership/inspectOwnership.ts

```diff
@@ -0,0 +1,37 @@
+import { Result } from '@chainlink/gauntlet-core'
+import { assertions, logger } from '@chainlink/gauntlet-core/dist/utils'
+import { SolanaCommand, TransactionResponse } from '@chainlink/gauntlet-solana'
+import { PublicKey } from '@solana/web3.js'
+import { CONTRACT_LIST, getContract } from '../../../lib/contracts'
+import { SolanaConstructor } from '../../../lib/types'
+
+export const makeInspectOwnershipCommand = (contractId: CONTRACT_LIST): SolanaConstructor => {
+  return class InspectOwnership extends SolanaCommand {
+    static id = `${contractId}:inspect_ownership`
+    static category = contractId
+
+    static examples = [`yarn gauntlet ${contractId}:inspect_ownership --network=devnet [CONTRACT_ADDRESS]`]
+
+    constructor(flags, args) {
+      super(flags, args)
+    }
+
+    execute = async () => {
+      // Assert that only one argument was inputted
+      assertions.assert(this.args.length == 1, `Expected 1 argument, got ${this.args.length}`)
+      // Get contract state
+      logger.info('Checking owner of ' + this.args[0])
+      const contract = getContract(contractId, '')
+      const program = this.loadProgram(contract.idl, contract.programId.toString())
+      const state = new PublicKey(this.args[0])
+      const onChainState = await program.account.state.fetch(state)
```

Comment:
The way to fetch the state changes for each contract unfortunately:
- OCR: `program.account.state.fetch(state)`
- AccessController: `program.account.accessController.fetch(state)`
- Store: 
  - Store: `program.account.store.fetch(state)`
  - Transmissions: `program.account.transmissions.fetch(state)`

Reply:
Any suggestions for differentiating store vs transmission? I currently use a switch statement based on contractId so store and transmission contracts would have the same id. I could add in a new CONTRACT_LIST enum for this use case.